src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/matmul_weights_decompression.cpp

```diff
@@ -38,43 +37,7 @@ namespace test {
  *               |
  *              Bias
  */
-
-enum class DecompressionSubtractType {
-    empty,  // no decompression subtract
-    scalar, // decompression subtract with scalar shape
-    full    // decompression subtract with per-channel or grouped shape
-};
-
-inline std::ostream& operator<<(std::ostream & os, DecompressionSubtractType type) {
-    switch (type) {
-        case DecompressionSubtractType::empty:
-            os << "empty";
-            break;
-        case DecompressionSubtractType::scalar:
-            os << "scalar";
-            break;
-        case DecompressionSubtractType::full:
-            os << "full";
-            break;
-        default:
-            OPENVINO_THROW("Not supported type for DecompressionSubtractType");
-    }
-    return os;
-}
-
-struct ShapeParams {
-    ShapeParams() = default;
-    ShapeParams(InputShape data_shape, ov::Shape weights_shape, int weights_group_size = -1)
-        : data_shape(std::move(data_shape)),
-          weights_shape(std::move(weights_shape)),
-          weights_group_size(weights_group_size) {}
-
-    InputShape data_shape;
-    ov::Shape weights_shape;
-    // Decompression group size. If the value is equal to -1, ordinary decompression is used
-    int weights_group_size;
-};
```

Comment:
This code is moved to weights_decompression_builders.hpp, `ShapeParams` renamed to `MatMulDecompressionShapeParams`